nucliadb/nucliadb/export_import/datamanager.py

```diff
@@ -130,6 +134,27 @@ def _get_storage_field(
             storage=self.storage, bucket=bucket, fullkey=key, field=cf
         )
 
+    async def delete_import(self, kbid: str, import_id: str):
+        key = STORAGE_IMPORT_KEY.format(import_id=import_id)
+        bucket = self.storage.get_bucket_name(kbid)
+        await self.storage.delete_upload(key, bucket_name=bucket)
+
+    async def delete_export(self, kbid: str, export_id: str):
+        key = STORAGE_EXPORT_KEY.format(export_id=export_id)
+        bucket = self.storage.get_bucket_name(kbid)
+        await self.storage.delete_upload(key, bucket_name=bucket)
+
+    async def try_delete_from_storage(self, type: str, kbid: str, id: str):
+        assert type in ("export", "import")
```

Comment:
nit: it's better to have a real exception in code and keep asserts in tests. They will even be ignored when running Python in certain O modes.